docs/source/manual/normalizers.rst

```diff
@@ -0,0 +1,22 @@
+===========
+Normalizers
+===========
+
+Burp Suite
+==========
+
+Uploading output from BurpSuite can only be done with paid versions (Professional). The community version doesn't have the required export functionality.
+
+The `official BurpSuite documentation for this functionality can be found here: <https://portswigger.net/burp/documentation/desktop/getting-started/generate-reports>`_.
+
+In general the approach is:
+
+- Under Target > Site map, select all objects/hosts you wish to export.
+- Right click and select Issues > Report issues for this host.
+- Follow the wizard. Make sure you export the file to XML.
+- Browse to your OpenKAT install.
+- Click on Objects > Upload raw file.
+- Select the burp raw file. As mime-type use `xml/burp-export`.
```

Comment:
Users can also navigate to the BurpSuite normalizer in their Katalogus and click upload on the normalizer detail page. the mime-type will then be pre-filled.

Reply:
There is no upload button on that page, only a disable button. You can click on the `xml/burp-export` link on that page, which redirects to the same upload page. So I'll add that as another option.